gabrielaEnache/src/test/java/com/endava/generics/BookTest.java

```diff
@@ -0,0 +1,28 @@
+package com.endava.generics;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class BookTest {
+    private Book book;
+
+    @Before
+    public void setUp() {
+        book = new Book("Java", "Oracle", 90f, 555);
+    }
+
+    @Test
+    public void setPositivePages() {
+        book.setPages(1000);
+        assertEquals(String.format("Page number is %d", 1000), 1000, book.getPages());
```

Comment:
no need to specify proper value in assert message, you already have expected and actual

Reply:
Fixed